Microsoft.Toolkit.Uwp.UI.Animations/Enums/EasingType.cs

```diff
@@ -0,0 +1,67 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Toolkit.Uwp.UI.Animations
+{
+    /// <summary>
+    /// Indicates how the animation interpolates between keyframes.
+    /// </summary>
+    public enum EasingType
```

Comment:
This one existed before, so not sure how we lost the move?

Reply:
Ooof, I must've accidentally moved this from within VS instead of manually doing `git mv` from the cmd 🤦‍♂️
VS has had a bug for years where it doesn't do a git rename and just deletes + recreates files when you rename or move them from the solution explorer, it's very annoying. There's an open issue about this [here](https://developercommunity.visualstudio.com/idea/942071/renaming-a-file-does-not-perform-a-git-rename.html), but still no updates on that. I usually do all the moves in terminal but I guess for this one I either slipped, or it might've been in a commit I had temporarily stashed/reverted while doing experimenting, which also causes VS to undo your `git mv` and just reinsert the delete/recreate on its own for some reason 😥